blog/2024/announcement-keycloak-organizations.adoc

```diff
@@ -0,0 +1,231 @@
+:title: Announcement: Support for Customer Identity and Access Management (CIAM) and Multi-tenancy
+:date: 2024-06-05
+:publish: true
+:author: Pedro Igor
+
+Dear Keycloak community,
+
+Thanks to the collaborative work with a lot of folks from the community and Red Hat's IT, we are delivering in Keycloak 25 the Keycloak Organizations
+feature.
+
+We are pleased to announce the beginning of a long journey to support Customer Identity and Access Management (CIAM) and,
+at some degree, also support for multi-tenancy when a realm needs to integrate with third parties such as customers and business
+partners.
+
+Keycloak Organizations is a feature that leverages the existing Identity and Access Management (IAM)
+capabilities of Keycloak to address CIAM uses cases like Business-to-Business (B2B) and Business-to-Business-to-Customer (B2B2C)
+integrations. By leveraging the existing capabilities available from a realm, the first release of this feature provides
+the very core capabilities to allow a realm to integrate with business partners and customers:
+
+* Manage Organizations
+* Manage Organization Members
+* Onboard members using different strategies such as invitation links and brokering
+* Decorate tokens with additional metadata about the organization that the subject belongs to
+
+The feature is being delivered initially as a technology preview feature with the ultimate goal to make it supported in Keycloak 26.
+There are many more capabilities in the https://github.com/keycloak/keycloak/issues/30180[roadmap] for this feature, and we consider this initial set of capabilities the very
+core of the feature that will allow us to build more capabilities on top. For this reason, your feedback is very important
+to make sure we are on the right path for solving real use cases around CIAM.
+
+For more details about the feature, consider reading the documentation available at the
+https://www.keycloak.org/docs/latest/server_admin/#assembly-managing-organizations_server_administration_guide[official documentation].
+
+== Getting started
+
+The Keycloak Organizations feature introduces changes on how users authenticate to a realm to identify whether a user is authenticating
+in the scope of an organization or the realm.
+
+One of the key changes introduced by the feature in terms of authentication is the introduction of an identity-fist login flow whenever
+you are authenticating to a realm that has the feature enabled.
```

Comment:
To me it might be a bit confusing to say that feature should be enabled for a realm, considering the fact that feature is enabled server-wide. Feel free to use any wording, my suggestion would be something like:
```suggestion
One of the key changes introduced by the feature in terms of authentication is the introduction of an identity-fist login flow whenever
you are authenticating to a realm that has the organization capability enabled.
```